app/shop.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import annotations
+
+
+class Shop:
+
+    def __init__(
+            self,
+            shop: dict) -> None:
+        self._name = shop["name"]
+        self.location = shop["location"]
+        self.products = shop["products"]
+
+    def customer_buying(
+            self,
+            customer: callable) -> None:
+
+        total_cost = 0
+
+        # print("Date: ", datetime.now().strftime("%d/%m/%y %H:%M:%S"))
+        print("Date: 04/01/2021 12:33:41")

```

Comment:
Use `datetime.now()` instead of hardcode

Reply:
Understand, but how to write it correct if task force me to print exactly 04/01/2021 12:33:41?